### PR TITLE
CLI: Implement create_keypair

### LINKED ISSUE #376: CLI: separate platform-specific commands
Currently, we have AquaCli available for both `.jvm` and `.js` targets. As a result, we have mocks in `.jvm` that just throw `???` (unimplemented).

I suggest moving AquaCli to `.js`. That would match the reality we have: `aqua` cli is available only for JS target. 

Also that will spare us from creating meaningless mocks.

### PR DESCRIPTION
**Pull request by folex**
fixes #376 

```
> node index.js "create_keypair"

{
  "peerId": "12D3KooWHL7ucaqeCqc4AQ4aWe7g7YiT8Wyv4jYDU4RxX2L6qx6o",
  "secretKey": "Y2pUnf5n9eIHofJHSclIMeLop0yLUYbFbq/g/XVvdmA=",
  "publicKey": "CAESIG+hG6JbYyGXmXM2EMobVm/srU5OikE+EG4oQTwBXoP8"
}
```

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -1,4 +1,6 @@
-version = 2.7.5
+version = 3.1.2
+
+runner.dialect = scala3
```
Comment (folex) [inline .scalafmt.conf:3]: Not sure about this, but VSCode demanded it

File: cli/.jvm/src/main/scala/aqua/KeyPair.scala [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import scala.concurrent.ExecutionContext
+import com.monovore.decline.Command
+import cats.effect.ExitCode
+
+object KeyPairOpts {
+    def createKeypair[F[_]](implicit ec: ExecutionContext): Command[F[ExitCode]] =
+        Command(
+            name = "create_keypair",
+            header = "Create a new keypair"
+        ) { ??? }
```
Comment (alari) [inline cli/.jvm/src/main/scala/aqua/KeyPair.scala:12]: Can you return the error ExitCode instead?

File: build.sbt [hunk 3]
```diff
@@ -51,14 +51,19 @@ lazy val cli = crossProject(JSPlatform, JVMPlatform)
       "com.monovore"  %%% "decline"        % declineV,
       "com.monovore"  %%% "decline-effect" % declineV,
       "co.fs2"        %%% "fs2-io"         % fs2V
-    )
+    ),
```
Comment (alari) [inline build.sbt:54]: This comma doesn't seem necessary

File: cli/.js/src/main/scala/aqua/KeyPairStringify.scala [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package aqua
+
+import scala.scalajs.js.JSON
+import scala.scalajs.js
+
+object KeyPairStringify {
+  def stringify(keypair: KeyPair): String = {
+    import java.util.Base64
```
Comment (alari) [inline cli/.js/src/main/scala/aqua/KeyPairStringify.scala:8]: Move this import to the top?

File: cli/src/main/scala/aqua/AquaCli.scala [hunk 5]
```diff
@@ -47,7 +47,7 @@ object AquaCli extends IOApp with Logging {
     implicit val aio: AquaIO[F] = new AquaFilesIO[F]
     implicit val ec = r.compute
 
-    runOpt orElse versionOpt
+    createKeypairOpt orElse runOpt orElse versionOpt
```
Comment (alari) [inline cli/src/main/scala/aqua/AquaCli.scala:50]: Maybe we can bundle the js-specific opts into `jsOpts` or smth like that, to see the clear difference of what depends on the platform and what doesn't.

File: npm/package.json [hunk 6]
```diff
@@ -14,7 +14,8 @@
     "aqua-cli": "error.js"
   },
   "scripts": {
-    "run": "node index.js"
+    "run": "node index.js",
+    "scala": "cp ../cli/.js/target/scala-3.0.2/cli-opt/main.js ./aqua.js && npm run run"
```
Comment (alari) [inline npm/package.json:18]: The name is not descriptive

File: cli/src/main/scala/aqua/AquaCli.scala (hunk 5 above)
Comment (folex) [reply cli/src/main/scala/aqua/AquaCli.scala:50]: Yeah, that's what I was coming at with #376

Comment (folex) [reply cli/src/main/scala/aqua/AquaCli.scala:50]: I think I'm not accustomed to Scala devtools enough to do this. We've discussed it with @DieMyst: he will do it in a separate PR.

File: npm/package.json (hunk 6 above)
Comment (folex) [reply npm/package.json:18]: `from:scala`?

File: cli/src/main/scala/aqua/AquaCli.scala (hunk 5 above)
Comment (folex) [reply cli/src/main/scala/aqua/AquaCli.scala]: Done

**Review by DieMyst — APPROVED** [review]

**Pull request merged**